test/repl.jl

```diff
@@ -386,7 +420,7 @@ temp_pkg_dir() do project_path; cd(project_path) do
         pkg"add Example"
         c, r = test_complete("rm Exam")
         @test "Example" in c
-        c, r = test_complete("add --man")
+        c, r = test_complete("up --man")
```

Comment:
Why was this test changed?

Reply:
Completions depend on the command. `add` does not have a `--manifest` option, so it will not complete. `up` does have a `--manifest` option

Reply:
Cool!

Reply:
Glad you think so :grin: